lib/js/components/ProgressDonutChart/ProgressDonutChart.vue

```diff
@@ -0,0 +1,287 @@
+	'primaryGhost': $color-primary-data-ghost,
+	'neutralMedium': $color-neutral-data-medium,
+	'neutral': $color-neutral-data,
+	'neutralWeak': $color-neutral-data-weak,
+	'neutralGhost': $color-neutral-data-ghost,
+	'infoMedium': $color-info-data-medium,
+	'info': $color-info-data,
+	'infoWeak': $color-info-data-weak,
+	'infoGhost': $color-info-data-ghost,
+	'successMedium': $color-success-data-medium,
+	'success': $color-success-data,
+	'successWeak': $color-success-data-weak,
+	'successGhost': $color-success-data-ghost,
+	'warningMedium': $color-warning-data-medium,
+	'warning': $color-warning-data,
+	'warningWeak': $color-warning-data-weak,
+	'warningGhost': $color-warning-data-ghost,
+	'failMedium': $color-fail-data-medium,
+	'fail': $color-fail-data,
+	'failWeak': $color-fail-data-weak,
+	'failGhost': $color-fail-data-ghost,
+);
+
+@keyframes dot-flashing {
+	0% {
+		background-color: $color-neutral-text;
+	}
+
+	50%,
+	100% {
+		background-color: transparent;
+	}
+}
+
+.progressDonutChart {
+	position: relative;
+	width: $progress-donut-chart-width;
+	height: $progress-donut-chart-height;
+	padding: 0;
+	margin: 0;
+	overflow: hidden;
+	box-sizing: border-box;
+
+	&__svg {
+		display: block;
+		width: $progress-donut-chart-width;
+		height: $progress-donut-chart-height;
+	}
+
+	&__circle {
+		fill: none;
+		stroke-width: $progress-donut-chart-circle-stroke-width;
+		// Disabled for v3.0
+		//stroke-linecap: round;
+		//stroke-linejoin: round;
+		transition: all $default-transition-time ease-out;
+		transform: rotate(90deg);
+		transform-origin: 50% 50%;
+	}
+
+	&__thumb {
+		stroke: $color-neutral-data-ghost;
+	}
+
+	&__track {
+		@each $class, $color-name in $progress-donut-chart-range-colors {
+			&.-#{$class} {
+				stroke: $color-name;
+			}
+		}
+
+		stroke-dasharray: $circle-circumference;
+		stroke-dashoffset: calc(
+			#{$circle-circumference} - (#{$circle-circumference} * (var(--length, 0) / 100))
+		);
+	}
+
+	&__loader {
+		stroke-dasharray: $circle-circumference;
+		stroke: $color-neutral-background-strong;
+		stroke-dashoffset: #{$circle-circumference - ($circle-circumference * (25 / 100))};
+		animation: a-spinAround 2s infinite linear;
+		transform: rotate(0deg);
+	}
+
+	&__label {
+		@include info-m-default-bold;
+
+		display: flex;
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		justify-content: center;
+		flex-wrap: nowrap;
+		align-items: center;
+		color: $color-neutral-text;
+	}
+
+	&__icon {
+		color: $color-success-text;
+	}
+
+	&__labelText {
+		max-width: 80%;
+		overflow: hidden;
+
+		&.-success {
+			color: $color-success-text;
+		}
+	}
+
+	&__loaderText {
+		position: relative;
+		width: $progress-donut-chart-loading-dot-width;
+		height: $progress-donut-chart-loading-dot-height;
+		margin-top: $space-xxxs;
+		border-radius: 100%;
+		background-color: $color-neutral-text;
+		color: $color-neutral-text;
+		animation: dot-flashing 1s infinite linear alternate;
```

Comment:
IMO, this animation should be faster

Reply:
[`72ec278` (#274)](https://github.com/bethinkpl/design-system/pull/274/commits/72ec278a74c1d268b6cbb5fa8606f72dcfa0b9c3)